server/plugin/plugin_test.go

```diff
@@ -71,7 +71,6 @@ func TestPluginOnActivate(t *testing.T) {
 				m := semver.MustParse(manifest.MinServerVersion)
```

Comment:
This test case ("greater minor version than minimumServerVersion") seems to be same as next test case. Would you remove this case?

Reply:
Done